shiny/ui/_theme_brand.py

```diff
@@ -368,167 +313,37 @@ def _prepare_typography_vars(brand: "Brand") -> dict[str, str]:
         )
 
         for field, prop in brand_typography.items():
-            if field not in typography_map:
-                warn_or_raise_unmapped_variable(f"typography.{field}")
-                continue
-
             for prop_key, prop_value in prop.items():
-                if prop_key in typography_map[field]:
-                    typo_sass_vars = typography_map[field][prop_key]
-                    for typo_sass_var in typo_sass_vars:
-                        mapped[typo_sass_var] = prop_value
-                else:
-                    warn_or_raise_unmapped_variable(f"typography.{field}.{prop_key}")
+                field = field.replace("-", "_")
+                prop_key = prop_key.replace("-", "_")
```

Comment:
Not sure if I follow 100%, but just noting that it seems those could create a variable name like `$brand_typography_headings_line_height`, when the Sass wants `$brand_typography_headings_line-height`?

Reply:
Great catch, thanks. You're right, I don't want these.